gamemodes/terrortown/gamemode/client/cl_player_ext.lua

```diff
@@ -252,7 +252,13 @@ function GM:SetupMove(ply, mv, cmd)
         return
     end
 
-    ply.isReady = true
+    -- we make the assumption that every player, that already is connected
+    -- is ready. We can't tell for sure, but this is the best we can do here
+    local plys = player.GetAll()
+
+    for i = 1, #plys do
+        plys[i].isReady = true
+    end
```

Comment:
(no action needed, just me rambling)

I see why this assumption is necessary, but i don't like that this might be wrong for quite some time depending on the circumstances.

Given that i don't know if a more accurate (but more complex) system would give us any benefits i'm fine with it.

Reply:
I thought about this for a while and I'm not sure if there is any good solution for that. Then again, the usecase for IsReady on the client is almost always for the client itself. And for the client the state is reliable.

Moreover it is 100% reliable on the server as well and this is where it matters the most. Therefore I think this issue is neglectable.